From 870135e4ed7f7b23755863c44ff1c636429388aa Mon Sep 17 00:00:00 2001
From: Martin Blumenstingl <martin.blumenstingl@googlemail.com>
Date: Fri, 1 May 2020 23:23:49 +0200
Subject: [PATCH 079/122] dt-bindings: clock: meson8b: Export the Meson8
 Ethernet (RMII) clock

Export CLKID_ETH_CLK because it is used as input for the Ethernet
controller on Meson8 SoCs.

Signed-off-by: Martin Blumenstingl <martin.blumenstingl@googlemail.com>
---
 include/dt-bindings/clock/meson8b-clkc.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/dt-bindings/clock/meson8b-clkc.h b/include/dt-bindings/clock/meson8b-clkc.h
index 78aa07fd7cc0..66e4569500a4 100644
--- a/include/dt-bindings/clock/meson8b-clkc.h
+++ b/include/dt-bindings/clock/meson8b-clkc.h
@@ -124,5 +124,6 @@
 #define CLKID_CTS_AMCLK		209
 #define CLKID_CTS_MCLK_I958	212
 #define CLKID_CTS_I958		213
+#define CLKID_ETH_CLK		222
 
 #endif /* __MESON8B_CLKC_H */
-- 
2.25.1

